content/micropython/00.first-steps/03.runtime-package/runtime-package.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "Arduino MicroPython Runtime"
+description: "Learn how to use the Arduino MicroPython runtime package, which allows you to write MicroPython code in a familiar Arduino style."
```

Comment:
change: "Arduino Runtime for MicroPython"

suggestion: "[...] familiar Arduino style and adds a few helpers"